DQM/SiPixelPhase1Common/src/SiPixelCoordinates.cc

```diff
@@ -0,0 +1,916 @@
+// -*- C++ -*-
```

Comment:
@davidlange6 Would you be more happy with `DataFormats/​SiPixelDetId`? I could prepare a commit for that. Also, more changes are queuing up in https://github.com/schneiml/cmssw/commits/phase1pixeldqm-post-christmas-version-on-pre2 , are you fine with sneaking some of them in here?